ProcessLib/HydroMechanics/HydroMechanicsFEM-impl.h

```diff
@@ -269,11 +302,11 @@ void HydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
 
     // pressure equation
     local_rhs.template segment<pressure_size>(pressure_index).noalias() -=
-        laplace_p * p + storage_p * p_dot + Kpu * u_dot;
+        laplace_p * (p - p_neq) + storage_p * p_dot + Kpu * u_dot;
```

Comment:
I think that every simulation starts from the previous status, whether which is equilibrium (may from the previous simulation) or non-equilibrium (specfied). Therefore p_neq should be p0, and local_rhs should not be changed.  This is a suggestion for future change. The PR itself is OK with the current consideration.